nginx/includes/redirects.conf

```diff
@@ -104,7 +104,8 @@ rewrite ^/docs/category/hosted-deposits-and-withdrawals$ "/platforms/anchor-plat
 rewrite ^/docs/category/build-a-wallet$ "/docs/category/build-a-wallet-with-the-wallet-sdk" permanent;
 rewrite ^/dapps "/docs/learn/interactive" permanent;
 rewrite ^/docs/learn/interactive/dapps "/docs/learn/interactive" permanent;
-rewrite ^/docs/learn/networks(.*)$ "/docs/learn/fundamentals/networks$1" permanent;
+rewrite ^/docs/learn/networks$ "/docs/networks/" permanent;
+rewrite ^/docs/learn/fundamentals/networks$ "/docs/networks/" permanent;
```

Comment:
i don't think it would really affect anything, but let's leave the trailing slash off the destination part:

`... "/docs/networks" permanent;`